holoviews/core/data/__init__.py

```diff
@@ -210,7 +208,7 @@ class Dataset(Element):
 
     def __init__(self, data, kdims=None, vdims=None, **kwargs):
         if isinstance(data, Element):
-            pvals = util.get_param_values(data)
+            pvals = get_param_values(data)
```

Comment:
Why remove the namespace qualification? Does it affect the import times? (It shouldn't!)

Reply:
See my comment above, I can have another look at this but:

>I had some weird issues when importing from .. import util getting the wrong utilities, hence I did this.